src/sentry/tasks/store.py

```diff
@@ -459,9 +459,7 @@ def _do_process_event(
 
                 # XXX(markus): When datascrubbing is finally "totally stable", we might want
                 # to drop the event if it crashes to avoid saving PII
```

Comment:
Does this TODO need to stick around?

Reply:
This is about the safe_execute which I'd like to keep for now